src/widgets/barchart.rs

```diff
@@ -700,15 +685,12 @@ mod tests {
             .data(&[("foo", 1), ("bar", 2)])
             .bar_style(Style::new().red());
         widget.render(buffer.area, &mut buffer);
-        let mut expected = Buffer::with_lines(vec![
-            "  █            ",
-            "1 2            ",
-            "f b            ",
-        ]);
+        let mut expected =
+            Buffer::with_lines(["  █            ", "1 2            ", "f b            "]);
```

Comment:
The buffer width in a bunch of tests was chosen so that rustfmt can wrap nicely while maintaining a 2D view of what the buffer looks like visually. If that breaks here, then the buffer should be wider / taller / use a macro to ensure that this doesn't wrap.

(The intent behind this is that every rendering test looks visually correct at a glance)